src/registry.json

```diff
@@ -1948,6 +1954,130 @@
+        "description": "With Aleph Zero Wallet, you can send transaction in native asset of Aleph Zero network, and sign transactions and messages using your MetaMask wallet.",
+        "audits": [
+          {
+            "auditor": "Sayfer",
+            "report": "https://sayfer.io/audits/metamask-snap-audit-report-for-fio/"
+          }
+        ],
+        "category": "interoperability",
+        "support": {
+          "contact": "https://help.fio.net",
+          "knowledgeBase": "https://app.fio.net/metamask",
+          "faq": "https://app.fio.net/metamask"
+        },
+        "sourceCode": "https://github.com/fioprotocol/fio-wallet-snap"
+      },
+      "versions": {
+        "1.0.2": {
+          "checksum": "b6Z0SYw5x5yYEYu1TiguPRbRV/izLsjK0FFJ/+strwg="
+        }
+      }
+    },
+    "npm:defi-armor-snap": {
+      "id": "npm:defi-armor-snap",
+      "metadata": {
+        "name": "DeFi Armor",
+        "author": {
+          "name": "Eulith",
+          "website": "https://www.eulith.com/"
+        },
+        "website": "https://www.eulith.com/",
+        "summary": "Protect on-chain assets with full transaction simulation and a curated security policy.",
+        "description": "The DeFi Armor snap brings the transaction simulation and policy engine of our standalone DeFi Armor product to MetaMask. Every transaction is deep-simulated and checked against our curated, customizable security policy. Any transfer of ETH or ERC-20 assets will be detected, even if it is deep inside the call stack. Warnings are issued when a transaction fails the security policy, along with a detailed explanation of what went wrong.",
+        "audits": [
+          {
+            "auditor": "Cure53",
+            "report": "https://github.com/Eulith/eulith-metamask-snap/blob/master/audit-report.pdf"
+          }
+        ],
+        "category": "transaction insights",
+        "support": {
+          "contact": "mailto:snap-support@eulith.com",
+          "knowledgeBase": "https://docs.google.com/document/d/1ZA-sUHE0a8e1kViCfG3xM9hm3vpZh_ZNcZ0xP-9JpLs/edit",
+          "faq": "https://docs.google.com/document/d/1ZA-sUHE0a8e1kViCfG3xM9hm3vpZh_ZNcZ0xP-9JpLs/edit"
+        },
+        "sourceCode": "https://github.com/Eulith/eulith-metamask-snap"
+      },
+      "versions": {
+        "0.2.3": {
+          "checksum": "fd+wiJngXnDU/6Lt8tH7hQmyDTVLQsrVtQhyXqsFU60="
+        }
+      }
+    },
+    "npm:@gobob/bob-snap": {
+      "id": "npm:@gobob/bob-snap",
+      "metadata": {
+        "name": "BOB",
+        "author": {
+          "name": "BOB",
+          "website": "https://www.gobob.xyz/"
+        },
+        "website": "https://www.gobob.xyz/",
+        "summary": "The BOB Bitcoin Snap allows users to receive and transfer BTC as well as too inscribe and transfer ordinals including BRC20 and other inscription types.",
```

Comment:
"as well as **to** inscribe and transfer"

Reply:
Fixed.